client/src/modules/cohort/index.jsx

```diff
@@ -397,6 +483,40 @@ class Cohort extends PureComponent {
             </div>
           </div>
         )}
+        {idDialogForRemovedListVisible && (
+          <Dialog
+            cancelLabel="common.button.cohorts"
+            confirmLabel="common.button.restore"
+            hasPermissions={isOwner}
+            onCancel={this.handleRedirect}
+            onConfirm={
+              isArchived && !isDeleted ? this.handleRestoreCohort : null
+            }
+            pending={pendingForCohortRestoring}
+            title={formatMessage(
+              {
+                id: 'cohort.actions.not-available'
+              },
+              { name }
+            )}
+          >
+            <p>
+              <FormattedMessage
+                id={externalAction.messageId}
+                values={{ name, performer: externalAction.performer }}
+              />
+              {!isOwner && (
+                <Fragment>
+                  {' '}
```

Comment:
is this neccessary?

Reply:
We join two spans iN the dialog without this There is no space between them. I tried to use styles:
https://github.com/sandstreamdev/eoc/pull/363#discussion_r329534033